cosmos/templates/base.html

```diff
@@ -24,7 +24,7 @@
 
 <body>
     {% cms_toolbar %}
-    <div class="container">
+    <div class="container" style="background-color: #90cad5;">
```

Comment:
this could be moved onto a dedicated css file later on

Reply:
yeah, but this should be handled by the base.html pr as this is only a quick dirty fix, so it doesnt need to be set in the plugins themselves, in order to prevent it from being done twice.